packages/gitbook/src/components/DocumentView/File.tsx

```diff
@@ -21,7 +21,7 @@ export async function File(props: BlockProps<DocumentBlockFile>) {
     const contentType = getSimplifiedContentType(file.contentType);
 
     return (
-        <Caption {...props} wrapperStyle={[]}>
+        <Caption {...props}>
             <Link
                 href={file.downloadURL}
                 download={file.name}
```

Comment:
@SamyPesse I removed the styles and empty wrapperStyle that was passed to Caption in the Files block. As also mentioned in the PR description: "improves consistency by having the Files block adhere to the same bordering styles from Caption, it was using its own but these were a bit different"

Let me know if you added explicitly different bordering styles on the Files block on purpose, but if not, I think its nice to streamline these styles while i'm at it.